fs/localfs/local_fs_nonwindows.go

```diff
@@ -18,3 +18,14 @@ func platformSpecificOwnerInfo(fi os.FileInfo) fs.OwnerInfo {
 
 	return oi
 }
+
+func platformSpecificDeviceInfo(fi os.FileInfo, parentDevice *fs.DeviceInfo) fs.DeviceInfo {
```

Comment:
we don't need parent device here, can you remove it? I believe you'll be able to revert few other changes that were required to pass this.